Description: Remove empty example output directories.
 These were the culprits:
   * usr/share/doc/Macaulay2/AbstractToricVarieties/example-output/
   * usr/share/doc/Macaulay2/Browse/example-output/
   * usr/share/doc/Macaulay2/EllipticIntegrals/example-output/
   * usr/share/doc/Macaulay2/EngineTests/example-output/
   * usr/share/doc/Macaulay2/Licenses/example-output/
   * usr/share/doc/Macaulay2/Polymake/example-output/
   * usr/share/doc/Macaulay2/RandomCurves/example-output/
   * usr/share/doc/Macaulay2/SCSCP/example-output/
   * usr/share/doc/Macaulay2/SectionRing/example-output/
   * usr/share/doc/Macaulay2/Style/example-output/
   * usr/share/doc/Macaulay2/ToricTopology/example-output/
   * usr/share/doc/Macaulay2/Units/example-output/
Author: Doug Torrance <dtorrance@piedmont.edu>
Last-Update: 2020-05-13

--- a/M2/Macaulay2/m2/html.m2
+++ b/M2/Macaulay2/m2/html.m2
@@ -822,6 +822,10 @@
  	  if not opts.IgnoreExampleErrors 
 	  then if hadExampleError then error(toString numExampleErrors, " error(s) occurred running examples for package ", pkg#"pkgname");
 
+	  -- if no examples were generated, then remove the directory
+	  if length readDirectory exampleOutputDir == 2 then
+	  	  removeDirectory exampleOutputDir;
+
 	  -- process documentation
 	  rawkey := "raw documentation database";
 	  if verbose then stderr << "--processing documentation nodes..." << endl;
